README.md

```diff
@@ -1,145 +1,92 @@
-rfc.fit(data_x, data_y)
+To get started, create a Truss with the following terminal command:
 
-# Create the Truss (serializing & packaging model)
-tr = truss.create(rfc, target_directory="iris_rfc_truss")
-
-# Serve a prediction from the model
-tr.predict([[0, 0, 0, 0]])
+```
+truss init text-classification
 ```
 
-### Package your model
+This will create an empty Truss at `./text-classification`.
 
-The `truss.create()` command can be used with any supported framework:
+### Implement the model
 
-* [Hugging Face](https://truss.baseten.co/create/huggingface)
-* [LightGBM](https://truss.baseten.co/create/lightgbm)
-* [MLflow](https://truss.baseten.co/create/mlflow)
-* [PyTorch](https://truss.baseten.co/create/pytorch)
-* [scikit-learn](https://truss.baseten.co/create/sklearn)
-* [Tensorflow](https://truss.baseten.co/create/tensorflow)
-* [XGBoost](https://truss.baseten.co/create/xgboost)
+The model serving code goes in `./text-classification/model/model.py` in your newly created Truss.
 
-But in more complex cases, you can build a Truss manually for any model. Start with `truss init my_truss` and follow [this guide](https://truss.baseten.co/create/manual).
+```python
+from typing import List
+from transformers import pipeline
 
-### Serve your model locally
 
-Serving your model with Truss, on Docker, lets you interface with your model via HTTP requests. Start your model server with:
+class Model:
+    def __init__(self, **kwargs) -> None:
+        self._model = None
 
-```
-truss run-image iris_rfc_truss
+    def load(self):
+        self._model = pipeline("text-classification")
+
+    def predict(self, model_input: str) -> List:
+        return self._model(model_input)
 ```
 
-Then, as long as the container is running, you can invoke the model as an API as follows:
+There are two functions to implement:
 
-```
-curl -X POST http://127.0.0.1:8080/v1/models/model:predict -d '[[0, 0, 0, 0]]'
-```
+* `load()` runs once when the model is spun up and is responsible for setting `self._model`
+* `predict()` runs each time the model is invoked and handles the inference. It can use any JSON-serializable type as input and output.
 
-### Configure your model for deployment
+### Add model dependencies
 
-Truss is configurable to its core. Every Truss must include a file `config.yaml` in its root directory, which is automatically generated when the Truss is created. However, configuration is optional. Every configurable value has a sensible default, and a completely empty config file is valid.
+The pipeline model relies on Transformers and PyTorch. These dependencies must be specified in the Truss config.
 
-The Truss we generated above in the quickstart sample has a good example of a typical Truss config:
+In `./text-classification/config.yaml`, find the line `requirements`. Replace the empty list with:
 
 ```yaml
-model_framework: sklearn
-model_metadata:
-  model_binary_dir: model
-  supports_predict_proba: true
-python_version: py39
 requirements:
```

Comment:
would be cool if we can show a diff view here

Reply:
Learned some new markdown syntax. Only issue is it makes it super annoying to copy and paste, so I'm going to revert to a non-diff version.

<img width="1059" alt="Screenshot 2023-07-05 at 5 28 55 PM" src="https://github.com/basetenlabs/truss/assets/98474633/4bc3205d-b80a-4db3-9030-0b6ecced7bea">

Reply:
hmm. I feel the diff is cleaner even if the copy and paste doesn't work. You can probably actually disable copy and paste (at least you could with mkdocs)